chat/index.html

```diff
@@ -23,12 +23,22 @@
 </div>
 <div class="cb">
     <article class="grid-9">
-        <h2>Now accessible via Slack</h2>
+        <h2>Send and receive the same messages on IRC and Slack.</h2>
         <p>
-          IRC isn’t the most user-friendly chat system. <a href="https://slack.com">Slack</a> on the other hand is a very user-friendly chat system. We have set up an <a href="http://rauchg.com/slackin/" title="&raquo; Slackin">IRC to Slack bridge</a> (that we generously borrowed from Socket.io, thank you!).
+          IRC is an open and very common, but not the most user-friendly chat system. Slack on the other hand is a very user-friendly one. We have developed an <a href="https://github.com/hoodiehq/slack-irc-duplex" title="slack-irc-duplex on GitHub">IRC/Slack duplexer</a>, so you can choose whatever you personally prefer.
```

Comment:
The sentence is actually correct. The main sentence is: `IRC is an open and very common chat system`. There is a comma missing, which is probably what confused you. Solution:

` IRC is an open and very common, but not the most user-friendly, chat system.`

or

` IRC is an open and very common – but not the most user-friendly – chat system.`

but maybe we just shouldn't be so clever an just say

`IRC is open and very common, but Slack is more user-friendly.`

saving an entire sentence in the process ;)